src/doc/rustc/src/platform-support.md

```diff
@@ -255,6 +255,7 @@ target | std | host | notes
 `bpfel-unknown-none` | * |  | BPF (little endian)
 `hexagon-unknown-linux-musl` | ? |  |
 `i386-apple-ios` | ✓ |  | 32-bit x86 iOS
+[`i586-pc-nto-qnx700`](platform-support/nto-qnx.md) | * |  | 32-bit x86 QNX Neutrino 7.0 RTOS |
```

Comment:
This must be a plain vanilla target name here, no links allowed (I ran into the same issue myself)